### DISCUSSION TITLE
emulatorjs integration

### DISCUSSION THREAD

**Discussion by webysther** [rommapp/romm#287] [Ideas]
Hi, great project, I just looking for something like this, more precise to run my few old games on browser, for this [emulatorjs](https://github.com/linuxserver/emulatorjs) from LSIO is perfect and with a great frontend to browse games with trailer and so.

They have a admin that make simple put data about rom but dont work great, will be nice have API or some sort integration to make possible to use a full featured rom manager with playable games on browser.

**Comment by gantoine** [comment]
There's an open issue covering this already, let's keep the conversation there for now. https://github.com/zurdi15/romm/issues/95
